### PR TITLE
fix(ChatMessage): focus outline visible only during keyboard navigation 

### PR DESCRIPTION
**Pull request by kolaps33**
Fix that focus outline is visible only while user is in keyboard navigation.

### CODE REVIEW

**Review by jurokapsiar — APPROVED** [review]

**Review by sophieH29 — APPROVED** [review]

File: src/components/Chat/ChatMessage.tsx [hunk 1]
```diff
@@ -38,12 +40,18 @@ export interface ChatMessageProps
 
   /** Timestamp of the message. */
   timestamp?: ShorthandValue
+
+  onFocus?: ComponentEventHandler<ChatMessageProps>
```
Comment (layershifter) [inline src/components/Chat/ChatMessage.tsx:49]: Missing description of the prop.

File: src/components/Chat/ChatMessage.tsx [hunk 2]
```diff
@@ -38,12 +40,18 @@ export interface ChatMessageProps
...
+}
+
+export interface MessageState {
```
Comment (layershifter) [inline src/components/Chat/ChatMessage.tsx]: Should be `ChatMessageState`

**Review by layershifter — COMMENTED** [review]
Added small comments, please also add a record to `CHANGELOG.md`

File: src/components/Chat/ChatMessage.tsx (hunk 1 above)
Comment (kolaps33) [reply src/components/Chat/ChatMessage.tsx:49]: added :)

**Review by layershifter — APPROVED** [review]

**Pull request merged**